src/DataType/FloatHandler.php

```diff
@@ -10,13 +10,8 @@ class FloatHandler extends ScalarHandler
     /**
      * {@inheritdoc}
      */
-    protected $type = 'double';
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getDataType(): string
+    protected function getType(): string
     {
-        return 'float';
+        return 'double';
```

Comment:
would this not be a breaking change? Anything already encoded as `float` would not be unserializable anymore

Reply:
Same here as `getType` returns the 'old' `$this->type`